types/nightwatch/index.d.ts

```diff
@@ -3371,11 +3412,11 @@ export interface ClientCommands extends ChromiumClientCommands {
      *   }
      * }
      *
-     * @see windowRect
+     * @see https://nightwatchjs.org/api/getWindowPosition.html
      */
     getWindowPosition(
         callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<NightwatchPosition>) => void,
-    ): this;
+    ): Awaitable<this, NightwatchPosition>;
```

Comment:
Should be `WindowPosition`. We can use the ones starting with `Nightwatch` when it's not about the window but something else, like an element.